CodeApp/Views/MarkdownView.swift

```diff
@@ -11,17 +11,21 @@ import SwiftUI
 
 struct ChangeLogView: View {
     @Environment(\.presentationMode) var presentationMode
+
     var body: some View {
         NavigationView {
             SimpleMarkDownView(text: NSLocalizedString("Changelog.message", comment: ""))
-                .navigationBarTitle(NSLocalizedString("Release Notes", comment: ""))
-                .navigationBarTitleDisplayMode(.inline)
-                .navigationBarItems(
-                    trailing:
-                        Button(NSLocalizedString("Done", comment: "")) {
-                            self.presentationMode.wrappedValue.dismiss()
+                .navigationBarTitle(NSLocalizedString("Release Notes", comment: ""), displayMode: .inline)
```

Comment:
This is failing the lint. It can be simplified to `.navigationBarTitle("Release Notes", displayMode: .inline)`

Reply:
Same for Button(NSLocalizedString("Done", comment: "")).